lighthouse-core/gather/gatherers/accessibility.js

```diff
@@ -65,6 +65,15 @@ function runA11yChecks() {
         // avoid circular JSON concerns
         node.element = node.any = node.all = node.none = undefined;
       });
+
+      // Ensure errors can be serialized over the protocol
+      if (result.error instanceof Error) {
+        result.error = Object.assign({
```

Comment:
maybe don't leave this open ended with `Object.assign` since it goes into artifacts, but I don't feel strongly

Reply:
we'd lose `errorNode` and other debug data on the error object, which ironically is the only stuff coming through right now. seems good to keep it though I also don't feel *very* strongly

Reply:
> we'd lose `errorNode` and other debug data on the error object, which ironically is the only stuff coming through right now. seems good to keep it though I also don't feel very strongly

Ah yeah, I meant copy the ones we want (like `errorNodes`) explicitly.

Reply:
ah gotcha, yeah sure I'll copy errorNode